Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -965,6 +987,27 @@ void UpdateToolbarItems()
 					n.UpdateToolBarVisible();
 			}
 
+			void UpdateLargeTitles()
+			{
+				var page = Child;
+				if (page != null && Forms.IsiOS11OrNewer)
+				{
+					var largeTitleDisplayMode = page.OnThisPlatform().LargeTitleDisplay();
+					switch (largeTitleDisplayMode)
+					{
+						case LargeTitleDisplayMode.Always:
+							NavigationItem.LargeTitleDisplayMode = UINavigationItemLargeTitleDisplayMode.Always;
+							break;
+						case LargeTitleDisplayMode.Automatic:
```

Comment:
future proof your code by adding a `default:` here

Reply:
I wanted to but c#7 was giving me errors?! going to look in the morning